packages/sdk/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 8.0.1
+
+### Breaking changes
+
+- `getEmbeddedModuleSchema` now uses the module version to determine in which custom wasm sections to look for the schema.
+  It also no longer is `async` because there isn't any need for it to be so.
```

Comment:
I know this is not something you introduced, but I don't think we want this change.
This means we are blocking the event loop while compiling smart contracts, which I don't think we want.
I have added a comment in the code as well.